Documentation/guides/contributing.md

```diff
@@ -270,14 +364,54 @@ multiple entries.  If no output, this is just "None".
 """
 ```
 
-- Provide ample comments within a function or method so that a relatively intelligent reader can follow along with your algorithm. Short comments can follow at the end of a line, longer comments (or descriptions of the step or task about to be performed) should precede a block of code on the line(s) above it.
+- Provide ample comments within a function or method
+  so that a relatively intelligent reader can follow along with your algorithm.
+  Short comments can follow at the end of a line,
+  longer comments (or descriptions of the step or task about to be performed)
+  should precede a block of code on the line(s) above it.
+
+Finally, if you write a new model module,
+the HARK team asks that you also provide a short mathematical writeup of the model as a PDF.
+This document does not need
+to go into great detail about the solution method for the model or the functions and classes included in the module,
+merely specify the economic model and provide a summary of how it is solved.
+See [ConsumptionSavingModels.pdf](https://github.com/econ-ark/HARK/blob/master/Documentation/ConsumptionSavingModels.pdf) for an example of this.
+
+#### Contributing to documentation
+
+Econ-ARK's documentation is managed with [Sphinx](https://www.sphinx-doc.org/).
+The documentation is written in [reStructuredText](https://www.restructuredtext.org) and
+[MyST Markdown](https://myst-parser.readthedocs.io/en/latest/index.html).
+
+Contributing to documentation involves editing the file you want to change,
+and creating a pull request in the usual fashion above.
+All changes to documentation are automatically rendered and deployed to
+``docs.econ-ark.org`` by our automated infrastructure.
+
+To test your changes to the documentation locally, you can render as follows:
+
+1. Install the dependencies for the documentation:
+
+   ```bash
+   $ pip install -e .[doc]
+   $ sudo apt-get install --yes pandoc
+   ```
+2. Run `sphinx-build`:
+
+   ```bash
+   $ sphinx-build -M html Documentation HARK-docs -T -W
+   ```
+3. View the HTML files
 
-Finally, if you write a new model module, the HARK team asks that you also provide a short mathematical writeup of the model as a PDF. This document does not need to go into great detail about the solution method for the model or the functions and classes included in the module, merely specify the economic model and provide a summary of how it is solved. See [ConsumptionSavingModels.pdf](https://github.com/econ-ark/HARK/blob/master/Documentation/ConsumptionSavingModels.pdf) for an example of this.
+   ```bash
+   $ firefox HARK-docs/html/index.html
```

Comment:
Maybe we should just ask contributors to open the file `HARK-docs/html/index.html` instead?